examples/rbac-remote/server/k8s/server_resources.yaml

```diff
@@ -5,23 +5,23 @@ metadata:
   namespace: feast-dev
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
+kind: Role
 metadata:
-  name: feast-cluster-role
+  name: feast-role
```

Comment:
should probably add namespace here, same as above `ServiceAccount` object -
```yaml
  name: feast-role
  namespace: feast-dev
```

Reply:
I think we cannot: I tried both the same NS as the RoleBinding or another one, but when I tried to export the manifest it was removed. I guess it MUST be on the same NS.